### ISSUE TITLE
Date And Amount Fields In Account View Don't Align Vertically In macOS

### ISSUE THREAD

**Issue by 2BeeOrNot** [moneymanagerex/moneymanagerex#3137]
MMEX version:
 - [X] 1.3.7
 - [X] 1.3.6
 - [X] 1.3.5
 - [X[ 1.5.0

The date and amount fields don't align vertically in the account view in macOS. See the scree capture below from 1.5.0. The dates are left justified but the digits and separators don't align vertically - notice the ragged right side of the dates. The amount field is right justified but the digits, group character and decimal character don't align vertically. I think it goes back to 1.3.5.

In Windows it it better. The amount field looks OK. The date separators are off slightly but overall the date field doesn't have the ragged ends like in macOS.

Is it just me? A macOS quirk? Something that can be fixed?

![Mac DateNumber Format](https://user-images.githubusercontent.com/63317977/112544299-35f3b000-8d8d-11eb-8e24-0b94151f4b90.jpg)

The dates align vertically in the Transaction Report. The amounts too.
![TrxReportDates](https://user-images.githubusercontent.com/63317977/112544375-4efc6100-8d8d-11eb-8751-7f2d257d525b.jpg)

**Comment by whalley** [comment]
Now you are being picky... :-)

**Comment by 2BeeOrNot** [comment]
Yeah. I debated about this one. Maybe it was just happening for me. Maybe it's an easy fix.
Having spent too many years in finance and accounting numbers that don't line up drive me nuts. :=)

I'll submit a feature request for macOS users that's not so picky.  Maybe you're up to the challenge. :=)

**Comment by whalley** [comment]
Seems to be a font related issue, let’s see what can be done.
